src/styles/utilies/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$secondary-color: #616070;
+$button-color: #00acdc;
+
+$secondary-font-size: 10px;
+$secondary-font-weight: 400;
+
+$width-of-card: 200px;
+
+$width-of-img: 160px;
+$height-of-img: 134px;
+
+$width-of-buy-button: 167px;
+$height-of-buy-button: 40px;
```

Comment:
On the other hand, if you don't use a variable twice, I guess there's no need to make a variable for it, unless the variables are connected by proportions with each other, of course)